packages/beeq/src/components/page-title/bq-page-title.tsx

```diff
@@ -0,0 +1,126 @@
+import { Component, h, Prop } from '@stencil/core';
+
+/**
+ * @part wrapper - The wrapper container `<div>` of the element inside the shadow DOM
+ * @part back - The container `<div>` that wraps the page title back icon
+ * @part title - The container `<div>` that wraps the pate title content
+ * @part icon - The `<bq-icon>` element used to render a predefined icon size based on the page title (back, edit, download icon)
+ * @part actions - The container `<div>` element used to render a edit and download icons for page title
+ * @part sub-title - The container `<div>` that wraps the sub-title element
+ */
+@Component({
+  tag: 'bq-page-title',
+  styleUrl: './scss/bq-page-title.scss',
+  shadow: true,
+})
+export class BqPageTitle {
+  // Own Properties
+  // ====================
+
+  // Reference to host HTML element
+  // ===================================
+
+  // State() variables
+  // Inlined decorator, alphabetical order
+  // =======================================
+
+  // Public Property API
+  // ========================
+
+  // Prop lifecycle events
+  // =======================
+
+  /** If true, the page title back icon will be shown */
+  @Prop({ reflect: true }) showBackIcon: boolean;
+
+  /** If true, the page title back icon will be shown */
+  @Prop({ reflect: true }) showActionIcons: boolean;
+
+  /** If true, the sub title of page title will be shown */
+  @Prop({ reflect: true }) showSubTitle: boolean;
```

Comment:
Not sure if we need these properties.  For the actions, we don't need to support any specific action icon, we don't which actions projects will have and we cannot assume those rendered in the `actions` slot as the default. That said, I think that handling the slots and displaying them when they have content would be enough.